cannot find jdbc folder

### Check Ahead

- [x] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

- [x] I am willing to try to fix this bug myself.


### Ⅰ. Issue Description

I use Docker deployment versions: 2.5.0, 2.4.0, 2.3.0 all have this error. Do I need to additionally download the JDBC jar package and mount it inside the container? I checked the official website and didn't see any instructions about this.

### Ⅱ. Describe what happened

_No response_

### Ⅲ. Describe what you expected to happen

_No response_

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

_No response_

### Ⅴ. Anything else we need to know?

_No response_

### Ⅵ. Environment

_No response_

> I pulled the latest image, but it didn't fix anything; it's still showing "cannot find jdbc folder".

Okay. I see that the errors are still there. The issue must be caused by something else.

I will fix it. Thank you.

```
Personal

Containers
hungry_keldysh

hungry_keldysh
ba7160e44ed0
apache/seata-server:latest
STATUS
Exited (1) (0 seconds ago)


	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.getResourceToInject(CommonAnnotationBeanPostProcessor.java:677)

	at org.springframework.beans.factory.annotation.InjectionMetadata$InjectedElement.inject(InjectionMetadata.java:228)

	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:119)

	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessProperties(CommonAnnotationBeanPostProcessor.java:329)

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1431)

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619)

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)

	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:336)

	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)

	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:334)

	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:209)

	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955)

	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)

	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:591)

	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)

	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:409)

	at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)

	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1300)

	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1289)

	at org.apache.seata.server.ServerApplication.main(ServerApplication.java:30)

Caused by: org.apache.seata.common.loader.EnhancedServiceNotFoundException: not found service provider for : org.apache.seata.core.store.db.DataSourceProvider caused by java.lang.IllegalStateException: Extension instance(definition: org.apache.seata.common.loader.ExtensionDefinition@518a436d, class: interface org.apache.seata.core.store.db.DataSourceProvider)  could not be instantiated: cannot find jdbc folder

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.createNewExtension(EnhancedServiceLoader.java:496)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.getExtensionInstance(EnhancedServiceLoader.java:477)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.loadExtension(EnhancedServiceLoader.java:453)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.load(EnhancedServiceLoader.java:346)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.access$200(EnhancedServiceLoader.java:277)

	at org.apache.seata.common.loader.EnhancedServiceLoader.load(EnhancedServiceLoader.java:117)

	at org.apache.seata.common.loader.EnhancedServiceLoader.load(EnhancedServiceLoader.java:101)

	at org.apache.seata.server.storage.db.lock.DataBaseLockManager.init(DataBaseLockManager.java:50)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.initInstance(EnhancedServiceLoader.java:724)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.createNewExtension(EnhancedServiceLoader.java:492)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.getExtensionInstance(EnhancedServiceLoader.java:477)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.loadExtension(EnhancedServiceLoader.java:453)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.load(EnhancedServiceLoader.java:346)

	at org.apache.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.access$200(EnhancedServiceLoader.java:277)

	at org.apache.seata.common.loader.EnhancedServiceLoader.load(EnhancedServiceLoader.java:117)

	at org.apache.seata.common.loader.EnhancedServiceLoader.load(EnhancedServiceLoader.java:101)

	at org.apache.seata.server.lock.LockerManagerFactory.init(LockerManagerFactory.java:70)

	at org.apache.seata.server.lock.LockerManagerFactory.init(LockerManagerFactory.java:53)

	at org.apache.seata.server.lock.LockerManagerFactory.getLockManager(LockerManagerFactory.java:48)

	at org.apache.seata.server.console.impl.AbstractService.<init>(AbstractService.java:45)

	at org.apache.seata.server.console.impl.AbstractBranchService.<init>(AbstractBranchService.java:27)

	at org.apache.seata.server.console.impl.db.BranchSessionDBServiceImpl.<init>(BranchSessionDBServiceImpl.java:60)

	at org.apache.seata.server.console.impl.db.BranchSessionDBServiceImpl$$EnhancerBySpringCGLIB$$cd0dd82e.<init>(<generated>)

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)

	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)

	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)

	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)

	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:213)

	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:87)

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateBean(AbstractAutowireCapableBeanFactory.java:1326)

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232)

```

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -415,6 +415,7 @@
                         </entrypoint>
                         <environment>
                             <TZ>Asia/Shanghai</TZ>
+                            <LOADER_PATH>/seata-server/libs/</LOADER_PATH>
                         </environment>
                     </container>
                     <extraDirectories>

